### PR TITLE
test vale again

### PR DESCRIPTION
**Pull request by mirnawong1**
pr to test vale

the vale linting is very basic and runs the following rules on changed files (diff only):

- typos
- repeated words
- title case
- ignores [words](https://github.com/dbt-labs/docs.getdbt.com/pull/5623/files#diff-583d30a4fb43741308235815656d0662fcc9cd7600466ced583043fe004e0d3bR1-R29) like dbt, dbt cloud, dbt core, etc.
- rejects DBT capitalized
- latin abbreviation (this is wonky)
- ui elements not bolded
- Flesch-Kincaid grade level (%s) below 8

note: i think a few of them might not be needed (last two bullets for example).

### CODE REVIEW

**Comment by mirnawong1** [comment]
hyperlint review

**Comment by nghi-ly** [comment]
@mirnawong1 : i decided to not muddle your test with the things i wanted to check out so reverted my commits. apologies for the untidiness 🙃

File: website/docs/docs/introduction.md [hunk 1]
```diff
@@ -9,6 +9,16 @@ pagination_prev: null
 
 dbt compiles and runs your analytics code against your data platform, enabling you and your team to collaborate on a single source of truth for metrics, insights, and business definitions. This single source of truth, combined with the ability to define tests for your data, reduces errors when logic changes, and alerts you when issues arise.
 
+## Title Case Header
+
+The tech writer writes teh documentation. It is organized by the writer too recieve this program.
+
+I can't can't seem to understand teh main points e.g. etc. 
+
+
```
Comment (mirnawong1) [inline website/docs/docs/introduction.md:18]: ## About DBT Cloud AND Explorer

File: website/docs/docs/introduction.md [hunk 2]
```diff
@@ -9,6 +9,16 @@ pagination_prev: null
...
+## About DBT Cloud AND Explorer
```
Comment (mirnawong1) [inline website/docs/docs/introduction.md:19]:
```suggestion
## About DBT Cloud AND Explorer

This is a teh organised way


```

**Pull request closed without merging**